lib/mutant/ast/meta/const.rb

```diff
@@ -0,0 +1,24 @@
+module Mutant
+  module AST
+    # Node meta information mixin
+    module Meta
+
+      # Metadata for const nodes
+      class Const
+        include NamedChildren, Concord.new(:node), NodePredicates
+
+        children :base, :name
+
+        # Test if AST node is a top level constant
```

Comment:
The description is IMO wrong. Unless its a cbase you cannot statically know the node is referencing a toplevel constant.

The predicate IMO should be named `possible_toplevel?`. 

Counter example:

```
module Foo
  class String
  end

  String # const node that dynamically returns `Foo::String` but predicate returns `true`.
end